c/tskit/trees.c

```diff
@@ -489,7 +489,7 @@ tsk_treeseq_init(
 
     if (tsk_treeseq_get_time_units_length(self) == strlen(TSK_TIME_UNITS_UNCALIBRATED)
         && !strncmp(tsk_treeseq_get_time_units(self), TSK_TIME_UNITS_UNCALIBRATED,
-               strlen(TSK_TIME_UNITS_UNCALIBRATED))) {
```

Comment:
Looks like you've got the wrong version of clang-format here. See [here](https://tskit.dev/tskit/docs/stable/development.html?highlight=clang%20format#sec-development-c-requirements)

(There's a long paper trail of us complaining that clang-format has such terrible compatibility :vomiting_face: )

Reply:
Simplest to install 6 I think, we're not going to change this in the short term. Anyway, whatever version of clang-format we'd settle on would probably be slightly different to the version you're using and you'd need to change anyway.